cms/djangoapps/contentstore/views/course.py

```diff
@@ -898,8 +898,7 @@ def settings_handler(request, course_key_string):
                     ee_min_score_pct = request.json.get('entrance_exam_minimum_score_pct', None)
 
                     # If the entrance exam box on the settings screen has been checked,
-                    # and the course does not already have an entrance exam attached...
-                    if entrance_exam_enabled and not course_entrance_exam_present:
+                    if entrance_exam_enabled:
```

Comment:
@mattdrayer i update this part because it is not allowing a user to update score from UI. need to delete an entrance exam if user wants to update score.